README.md

```diff
@@ -1,6 +1,6 @@
 # shimming-toolbox
 
-[![Build Status](https://travis-ci.com/shimming-toolbox/shimming-toolbox.svg?branch=master)](https://travis-ci.com/shimming-toolbox/shimming-toolbox) [![Documentation Status](https://readthedocs.org/projects/shimming-toolbox/badge/?version=latest)](https://www.shimming-toolbox.org/en/latest/?badge=latest)
+[![Build Status](https://travis-ci.com/shimming-toolbox/shimming-toolbox.svg?branch=master)](https://travis-ci.com/shimming-toolbox/shimming-toolbox) [![Documentation Status](https://readthedocs.org/projects/shimming-toolbox-py/badge/?version=latest)](https://shimming-toolbox.org/en/latest/?badge=latest)
```

Comment:
shouldn’t we instead update the project name on RTD?

Reply:
It seems we can change the project name, but RTD only changes the URL slug on a case-by-case basis (see readthedocs/readthedocs.org#5403). This PR just fixes up the badge location referencing the old project name with the -py suffix.

Reply:
> should we submit a request?

yes let's do that